be/src/olap/rowset/segment_v2/vertical_segment_writer.cpp

```diff
@@ -368,24 +368,9 @@ Status VerticalSegmentWriter::_append_block_with_partial_content(RowsInBlock& da
 
     std::vector<RowsetSharedPtr> specified_rowsets;
     {
+        sleep(120);
```

Comment:
why sleep 120?

Reply:
This is just a draft. This sleep 120 will be added in debug point test.